dbt/adapters/redshift/relation.py

```diff
@@ -1,13 +1,92 @@
 from dataclasses import dataclass
-from dbt.adapters.postgres.relation import PostgresRelation
+from typing import Optional
+
+from dbt.adapters.base.relation import BaseRelation
+from dbt.adapters.relation_configs import (
+    RelationConfigChangeAction,
+    RelationResults,
+)
+from dbt.context.providers import RuntimeConfigObject
+from dbt.exceptions import DbtRuntimeError
+
+from dbt.adapters.redshift.relation_configs import (
+    RedshiftMaterializedViewConfig,
+    RedshiftMaterializedViewConfigChangeCollection,
+    RedshiftAutoRefreshConfigChange,
+    RedshiftBackupConfigChange,
+    RedshiftDistConfigChange,
+    RedshiftSortConfigChange,
+    RedshiftIncludePolicy,
+    RedshiftQuotePolicy,
+)
 
 
 @dataclass(frozen=True, eq=False, repr=False)
-class RedshiftRelation(PostgresRelation):
-    # Override the method in the Postgres Relation because Redshift allows
-    # longer names: "Be between 1 and 127 bytes in length, not including
-    # quotation marks for delimited identifiers."
-    #
-    # see: https://docs.aws.amazon.com/redshift/latest/dg/r_names.html
+class RedshiftRelation(BaseRelation):
+    include_policy = RedshiftIncludePolicy  # type: ignore
+    quote_policy = RedshiftQuotePolicy  # type: ignore
+
+    def __post_init__(self):
```

Comment:
This is the only remaining method in `PostgresRelation` that isn't in `BaseRelation`. Re-defining it and inheriting from `BaseRelation` is less error prone going forward as we now have different definitions for similar methods with materialized views. This also  contributes to a longer term goal of removing `dbt-postgres` as a dependency for `dbt-redshift`.